javascript/javascript3/week1/weatherApp.js

```diff
@@ -0,0 +1,171 @@
+// All Buttons from html
```

Comment:
Really nice job with the weather app! I think you have done a good job using functions and good naming to make your code easy to read. Very cool with the optional features as well 🚀 
I only have one or two main comments:
- You can simplify your code even further by making an API call function and a single display function that maybe takes a few arguments to make the function a little more flexible
- My opinion is to be a little careful with global variables, but it's fair if you disagree 😄 

Please let me know on slack if you have any questions about my comments 😄

Reply:
Thanks. I will update my codes with your advices 👍